### PR TITLE
0x7 - Add elements on Test Strategy

### PR DESCRIPTION
**Pull request by romualdszkudlarek**
plus cosmetic changes to previously authored text

### CODE REVIEW

File: Document/0x07-Security-Testing-SDLC.md [hunk 1]
```diff
@@ -86,25 +88,42 @@ The following diagram provides an overview of the way to perform test in the SDL
...
+* **Unit Tests**: unit tests make up the family of tests that are the closest to source code (e.g. they are focused on a single unit) as they are performed along with code. According to the methodology in use, they can be created either before code is developed or right after. Whatever the case, the end goal is to verify that procuced code is behaving as expected, but also that proper controls are put in place to prevent abuse cases (input filtering / validation, whitelisting, ...) and cannot be circumvented. Unit Tests are used to detect issues early in the development lifecycle to be able to fix them as soon and easily as possible. They are different from other forms of tests, like Integration / Verification / Validation tests, and may not be used to detect the same kind of issue. �Often, Unit Tests are aided with tools; a few of them are listed in another section.
```
Comment (sushi2k) [inline Document/0x07-Security-Testing-SDLC.md]: It seems sometimes are some unicode characters there that cannot be displayed, for example "�Often" and also in other areas of the PR. Can you please clean this?

File: Document/0x07-Security-Testing-SDLC.md [hunk 2]
```diff
@@ -86,25 +88,42 @@ The following diagram provides an overview of the way to perform test in the SDL
...
+Ideally, a Test Strategy addresses topics such as:
+- objectives to be met and description of risks to be put under control.
+- how these objectives will be met and risks reduced to an acceptable level: which tests will be mandatory, who will perform them, how, when, at which frequency.
+- acceptance criteria of the current project.
+
+In order to follow its effectiveness and progress, metrics should be defined, updated all along the project and periodically communicated. An entire book could be written on the relevant metrics to choose; the best that can be said is that they depend on risk profiles, projects and organizations. However, some examples can be given : metrics include 
+- the number of stories related to security controls that are implemented,
+- code coverage for unit tests on security controls and sensitive features,
+- the number of security bugs found by static analysis tools upon each build,
+- the trend of the backlog for security bugs (may be sorted by criticity).
```
Comment (sushi2k) [inline Document/0x07-Security-Testing-SDLC.md]: Is criticity used in the IT security context? Never heard it. Otherwise maybe use just criticality.

**Comment by romualdszkudlarek** [comment]
Ok, done with changes implementation.

**Comment by b-mueller** [comment]
Thanks @romualdszkudlarek

**Pull request merged**